src/com/richard/walker/weatherstation/vendor/StandardRegionalWeatherAdapter.java

```diff
@@ -0,0 +1,21 @@
+package com.richard.walker.weatherstation.vendor;
+
+import com.richard.walker.weatherstation.domain.WeatherStation;
+
+import java.util.List;
+
+public class StandardRegionalWeatherAdapter implements StandardRegionalWeather {
+
+    NoaaRegionalWeather noaaRegionalWeather;
+
+    public StandardRegionalWeatherAdapter(NoaaRegionalWeather noaaRegionalWeather) {
```

Comment:
I wanna add a comment here to show that we're using dependency injection. I think it'll be beneficial to remind people that the concept isn't limited to the ```@Inject ``` annotation.